src/PumpStream.php

```diff
@@ -34,20 +34,20 @@ class PumpStream implements StreamInterface
     private $buffer;
 
     /**
-     * @param callable $source  Source of the stream data. The callable MAY
-     *                          accept an integer argument used to control the
-     *                          amount of data to return. The callable MUST
-     *                          return a string when called, or false on error
-     *                          or EOF.
-     * @param array    $options Stream options:
-     *                          - metadata: Hash of metadata to use with stream.
-     *                          - size: Size of the stream, if known.
+     * @param callable                            $source  Source of the stream data. The callable MAY
```

Comment:
quite recently I learned that phstan can read this:

`callable(RequestInterface): ResponseInterface`

which validates that the callable is this:

`function (RequestInterface $request): ResponseInterface`

in this case maybe we should have `callable(int)`.. do you think it's worth it?

Reply:
cool, added it